sportsref/ncaaf/players.py

```diff
@@ -33,52 +33,46 @@ def getDoc(self):
     @sportsref.decorators.memoized
     def name(self):
         doc = self.getDoc()
-        name = doc('div#info_box h1:first').text()
+        name = doc('div#meta h1:first').text()
         return name
 
     @sportsref.decorators.memoized
     def position(self):
         doc = self.getDoc()
-        rawText = (doc('div#info_box p')
+        rawText = (doc('div#meta p')
                    .filter(lambda i,e: 'Position' in e.text_content())
                    .text())
-        rawPos = re.search(r'Position: (\S+)', rawText, re.I).group(1)
-        allPositions = rawPos.split('-')
-        # TODO: right now, returning just the primary position for those with
-        # multiple positions
+        rawPos = re.search(r'Position : (\S+)', rawText, re.I).group(1)
+        allPositions = rawPos.split('/')
+        # TODO: returning just the first position for those with
+        # multiple positions. Should return the last position played
```

Comment:
I was thinking that players are typically referenced on most websites by one position. That position is usually the last position played. I'm thinking of Terrelle Pryor, but don't know a ton of these examples. Peppers is a pretty good example with all those postions. I'm fine with a list as I can just parse out the last position played from annual data.